supabase/migrations/20240503124339_rename_request_deleted_and_request_denied.sql

```diff
@@ -0,0 +1,13 @@
+alter table "public"."events" drop constraint "events_new_parcel_status_fkey";
+
+alter table "public"."events" add constraint "public_events_new_parcel_status_fkey" FOREIGN KEY (new_parcel_status) REFERENCES status_order(event_name) ON UPDATE CASCADE not valid;
```

Comment:
fyi for Ricky - I told Kay to use on update cascade - i thought this would be good, but aware you might not think this is best

Reply:
Agree we should `on update cascade`!